app/models.py

```diff
@@ -33,12 +33,14 @@ class User(db.Model):
                            nullable=False)
     password_changed_at = db.Column(db.DateTime, index=False, unique=False,
                                     nullable=False)
+    role = db.Column(db.String, index=False, unique=False, nullable=False)
```

Comment:
Should this be defined as an [sqlalchemy.types.Enum](http://docs.sqlalchemy.org/en/latest/core/type_basics.html#sqlalchemy.types.Enum) given that we are applying the constraint in the migration?